### ISSUE TITLE
Get selected value(s) from a `select` component

### ISSUE THREAD

**Issue by TuxCoder** [ctron/patternfly-yew#37]
I my not quite sure, but I don't see the ability to get a reference to a `patternfly-yew::Select<K>`
eg. there is no `ref` / `r#ref` or something similar to used with `use_node()` hook.

setup is something like:
```rust

#[derive(Debug, Clone, PartialEq)]
struct Foobar {
    id: u32,
    name: String,
}

impl Display for Foobar {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        write!(f, "{}", self.name)
    }
}

#[function_component(Edit)]
pub fn edit(props: &Props) -> Html {
    let onsubmit ={
        Callback::from(move |e: SubmitEvent| {
          // get selected element
        }
    }
  }

  html! {
    <Form>
      <Select<Foobar>>
          ....
      </Select<Foobar>>
   </Form>
  }
}
```

how can I access the data from the `Select` component?

**Comment by TuxCoder** [comment]
Ok after looking to much time to the api, I found a way how to extract the data:

with the `variant` property I'm able to add a callback method to fetch the data, that I can then save in a local `use_state()` hook.

I think I see why you did this (type safety from the callback method) but would be nice to have an example in the get started. For this feature as it is a bit in intuitive with the name `variant`

eg. example:
```rust

#[function_component(Edit)]
pub fn edit(props: &Props) -> Html {
    let state  = use_state(|| None);

    let select_callback = {
        let state = state.clone();
         Callback::from(move |d: Foobar| {
            let sort = state.as_ref().unwrap().as_ref().unwrap();
            state.set(Some(d.id));
        })
    };
    let onsubmit ={
        let state = state.clone();
        Callback::from(move |e: SubmitEvent| {
          // use state.as_ref().......
        }
    }
  }

  html! {
    <Form onsubmit={onsubmit}>
      <Select<Foobar> variant={SelectVariant::Single(select_callback)}>
          ....
      </Select<Foobar>>
   </Form>
  }
}

**Comment by ctron** [comment]
You are right, this lacks documentation and a better example in the quickstart.

Would you be able to add an example, I would appreciate that.

**Comment by TuxCoder** [comment]
cloesing as fixed in [#16](https://github.com/ctron/patternfly-yew-quickstart/pull/16)
